jira7/jira_types.bal

```diff
@@ -35,20 +34,19 @@ public type ProjectSummary record {
     string projectTypeKey;
 };
 
-documentation{Represents a detailed jira project.
-    F{{resource_path}} API resource url
-    F{{id}} project Id
-    F{{key}} project key
-    F{{name}} project name
-    F{{description}} project description
-    F{{leadName}} jira username of the project lead
-    F{{projectTypeKey}} type of the project("software" or "business")
-    F{{avatarUrls}} project avatar urls
-    F{{projectCategory}} details of project category
-    F{{issueTypes}} support issue types of the project
-    F{{components}} summarized details about components of the project
-    F{{versions}} detatils of project versions
-}
+# Represents a detailed jira project.
+# + resource_path - API resource url
+# + id - project Id
+# + key - project key
+# + name - project name
+# + description - project description
+# + leadName - jira username of the project lead
+# + projectTypeKey - type of the project("software" or "business")
+# + avatarUrls - project avatar urls
+# + projectCategory - details of project category
+# + issueTypes - support issue types of the project
+# + components - summarized details about components of the project
+# + versions - detatils of project versions
```

Comment:
Again, inconsisten use of cases. jira -> Jira, url -> URL
And sentence case should be used (i.e., first letter upper case)